ci/Dockerfile

```diff
@@ -10,14 +10,15 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 ARG SKIP_BUILD=
 ARG ZEEK_LTS=1
-ARG ZEEK_VERSION=3.0.11-0
+ARG ZEEK_VERSION=4.0.2-0
 
 CMD ["sh"]
 ENV DEBIAN_FRONTEND=noninteractive
 ENV PATH="/opt/spicy/bin:/opt/zeek/bin:${PATH}"
 
 RUN apt-get update \
  && apt-get install -y --no-install-recommends curl ca-certificates gnupg2 less sudo \
+ && pip3 install --no-cache-dir pre-commit \
```

Comment:
If we want to execute `pip3` here we need to move the install of `python3-pip` (and maybe `python` and `python3` as well) from l.26 to the `apt-get install` invocation above.

Reply:
Fixed

Reply:
Are all these actually needed here? It looks like you moved also e.g., btest dependencies here. While this works this makes it quiet messy and hard to know which step introduced which dependency (and e.g., if we'd now drop the Zeek installation below we'd likely forget to update the list here).

But probably not something which _must_ be fixed now.